exercises/concept/windowing-system/.meta/exemplar.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+namespace windowing_system {
+
+struct size {
+    int width{80};
+    int height{60};
+    void resize(int newWidth, int newHeight);
+};
+
+struct position {
+    int x;
+    int y;
+    void move(int newX, int newY);
+};
+
+struct programWindow {
+    programWindow();
+    ~programWindow();
+    size* screenSize{nullptr};
+    size* windowSize{nullptr};
+    position* windowPosition;
```

Comment:
not that it matters, but why default values for `screenSize` and `windowSize` but not for `windowPosition`?